.devcontainer/Dockerfile

```diff
@@ -20,7 +20,14 @@ RUN apt-get update && apt-get -y install git procps jq \
     #&& apt-get install -y sudo \
     #&& echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME \
     #&& chmod 0440 /etc/sudoers.d/$USERNAME \
+    && curl -fsSL https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]')/gpg | apt-key add - 2>/dev/null \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/$(lsb_release -is | tr '[:upper:]' '[:lower:]') $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y docker-ce-cli \
     #
+    # Install Docker Compose
+    && curl -sSL "https://github.com/docker/compose/releases/download/1.24.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose \
```

Comment:
At some point should the version (in this case 1.24.0) be moved to a variable?

Reply:
Great idea.  We should probably parameterize all version numbers into a config file a la package.json, requirements.txt, etc.

That way, advanced users (and ourselves) can specify specific versions.  It also centralizes versions into one place to make it easy to update.